README.md

```diff
@@ -1 +1,73 @@
-# tic-tac-toe
\ No newline at end of file
+# Tic-tac-toe
+
+> This projects implements the tic-tac-toe game in ruby. Playable through terminal.
+
+Tic-tac-toe, or Xs and Os is a paper-and-pencil game for two players. The player who succeeds in placing three of their marks in a horizontal, vertical, or diagonal row is the winner.
+[Tic-tac-toe - Wikipedia](https://en.wikipedia.org/wiki/Tic-tac-toe)
+
+## Built With
+
+- Ruby
+- Rubocop (linter)
+
+## Getting started
+
+### Prerequisites
+- To run this project, you must have Ruby installed in your computer.
```

Comment:
- [ ] **[OPTIONAL]** A quick summary of how to install Ruby can be added if the user trying to play the game didn't have ruby installed.